OpenSim/Common/RegisterTypes_osimCommon.cpp

```diff
@@ -64,7 +64,7 @@ OSIMCOMMON_API void RegisterTypes_osimCommon()
 {
   try {
 
-    Object::registerType(Connector<Object>());
+    Object::registerType(Connector<Component>());
```

Comment:
We briefly discussed this a bit neither Connectors to base Object or Component make much sense. Object bothered me more so I changed it to Component, but perhaps neither should be here. If a user makes a utility object (deriving from Object) they can register a connector for it.